### DISCUSSION TITLE
Not updating PEPPOL Directory

### DISCUSSION THREAD

**Discussion by canilsenlogiq** [phax/phoss-smp#158] [Q&A] [answered]
Hello, need some help.

We have enabled the Peppol Directory in settings:
![PEPPOLDIR](https://user-images.githubusercontent.com/77612849/110933602-40459100-832d-11eb-860e-1723aa1ecef2.PNG)

Read: https://github.com/phax/phoss-smp/wiki/Peppol-Directory-Integration
And my understanding is that we should now only need to enable these settings in the SMP Settings.
However, we do not see any of our registrations in the Peppol Directory.

Note: I also cannot see these settings in our smp-server.properties file even if they are set in the GUI.

Am I missing something?

**Comment by phax** [comment, marked as answer]
Hi,
if you enabled these settings then the prerequisites are fulfilled.
The next step  for you is to create a "Business Card" for the participant. Only the "Business Cards are synced over to the Peppol Directory.

And related to the settings: you are correct, the `smp-server.properties` files only contains the "default settings", and this file is NEVER modified by the software - it is completely read-only.
Depending on the backend you chose (XML, SQL, MongoDB) the files reside either on disk in the file `smp-settings.xml`  (XML and SQL backend) or in a MongoDB table.

hth, Philip

**Reply by canilsenlogiq** [reply]
Thank you, it worked like a charm!
